CHANGELOG.md

```diff
@@ -0,0 +1,40 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/), and this project
+tries to adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+
+## [Unreleased]
+
+### Added
+
+- Started keeping a changelog. Added a changelog file, and added a changelog nagger to CI.
+
+### Changed
+
+### Deprecated
+
+### Removed
+
+### Fixed
+
+### Security
+
+## [1.2.2] - 2024-12-13
+
+### Fixed
+
+- Fixed a problem with `galois_group` [PR #4396](https://github.com/oscar-system/Oscar.jl/pull/4396)
```

Comment:
That's better. I've now adjusted https://github.com/oscar-system/Oscar.jl/pull/4396 to use this as title, and added the "use title" label.


We also should add labels reflecting the categories we use. While I am still not convinced that "Added" / "Changed" / etc. are what we want (after all the description usually makes that clear already: "Fixed a problem with ..."), I am OK with trying that out for now, we can then iterate.

But that would suggestion suitable labels. E.g. `category: change`, `category: fix`, `category: addition` (or any different prefix, like `kind:` or `type:`. Or no prefix, whatever. As long as there is a way to find out which labels are relevant. Having a common prefix simply makes that easier)